core/src/main/java/com/github/weisj/darklaf/DarkLaf.java

```diff
@@ -400,10 +425,9 @@ public void propertyChange(final PropertyChangeEvent evt) {
 
     @Override
     public boolean getSupportsWindowDecorations() {
-        return LafManager.isDecorationsEnabled()
+        return !"false".equals(System.getProperty("darklaf.decorations"))
+            && LafManager.isDecorationsEnabled()
             && LafManager.getTheme().useCustomDecorations()
             && Decorations.isCustomDecorationSupported();
```

Comment:
There are places which query only `Decorations.isCustomDecorationSupported()`, so it is not consistent.
Either `Decorations` should call `Darklaf.getSupportsWindowDecorations`, or all the conditions should be moved to `Decorations.isCustomDecorationSupported`

Reply:
I moved everything to `Decorations#isCustomDecorationSupported`.